modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-migration-process_{context}"]
+= Migrating from oc-mirror plugin v1 to oc-mirror plugin v2
+
+Migrating from oc-mirror v1 to v2 requires manually updating the ImageSetConfiguration, adjusting mirroring commands, and ensuring proper cleanup of v1 artifacts. Follow these steps to transition smoothly:
+
+. Modify the API version and remove deprecated fields in your `ImageSetConfiguration`. Refer below for the examples before and after the configuration:
++
+.Example output with oc-mirror plugin v1 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+storageConfig:  # REMOVE this field in v2
+  local:
+    path: /var/lib/oc-mirror
+----
++
+.Example output with oc-mirror plugin v2 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+----
+
+. Commands have changed to align with oc-mirror plugin v2's new workflow.
```

Comment:
This is not the proper place to introduce the v2 command , better to change to `Command line changes`